src/components/Inventory/InventoryTable.tsx

```diff
@@ -50,6 +53,22 @@ interface InventoryTableProps {
 
 export default function InventoryTable(props: InventoryTableProps): JSX.Element {
   const { organization, results, setTemporalSearchValue, temporalSearchValue, filters, setFilters } = props;
+  const [selectedRows, setSelectedRows] = useState<any[]>([]);
+  const history = useHistory();
+  const trackingEnabled = isEnabled('Tracking V1');
+
+  const withdrawInventory = () => {
+    const speciesIds = selectedRows.filter((row) => row.species_id).map((row) => `speciesId=${row.species_id}`);
+    if (!speciesIds.length) {
+      // we can't handle deleted inventory today
+      return;
+    }
+
+    history.push({
+      pathname: APP_PATHS.INVENTORY_WITHDRAW,
+      search: `?${speciesIds.join('&')}`,
```

Comment:
@constanzauanini just to illustrate - here's how I am doing this.

Reply:
Thanks!